### PR TITLE
Translation of 01-hello-world

### PR DESCRIPTION
**Pull request by christianhegedues**

### CODE REVIEW

**Comment by christianhegedues** [comment]
Thank you @mka4975 for the hint.
I changed the sentence grammatically and it should be fine for now.

File: 1-js/02-first-steps/01-hello-world/article.md [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
 
-If we have a lot of JavaScript code, we can put it into a separate file.
+Wenn wir viel JavaScript-Code haben, können wir diesen in eine separate Datei legen.
 
-Script files are attached to HTML with the `src` attribute:
+Skript-Dateien werden zu HTML mit dem `src`-Attribute hinzugefügt:
 
 ```html
-<script src="/path/to/script.js"></script>
+<script src="/pfad/zum/skript.js"></script>
```
Comment (JoeGab) [inline 1-js/02-first-steps/01-hello-world/article.md]:
```suggestion
<script src="/pfad/zum/script.js"></script>
```

Comment (1H0) [reply 1-js/02-first-steps/01-hello-world/article.md]: Good point! Please update this.. Then we're ready to merge 💃

Comment (JoeGab) [reply 1-js/02-first-steps/01-hello-world/article.md]: I am not allowed to update this, right? The autho mustr, right? Or can I, and just didn't realize?

Comment (1H0) [reply 1-js/02-first-steps/01-hello-world/article.md]: @JoeGab Christian has to implement the changes... Then we can review again and merge if everything is fine 👍. The Authomation doesn't mark this as Requested changes becauase you aren't a contributor rn.

**Pull request merged**